test/parallel/test-pipe-file-to-http.js

```diff
@@ -45,18 +45,18 @@ server.on('listening', function() {
 });
 
 function makeRequest() {
-  var req = http.request({
+  const req = http.request({
     port: server.address().port,
     path: '/',
     method: 'POST'
   });
 
-  var s = fs.ReadStream(filename);
+  const s = fs.ReadStream(filename);
   s.pipe(req);
-  s.on('close', function(err) {
+  s.on('close', common.mustCall((err) => {
     if (err) throw err;
     clientReqComplete = true;
```

Comment:
Because this is now in a `common.mustCall()`, you should be able to remove `clientReqComplete` from the test completely.